### PR TITLE
Combine index menu defs

### PR DESCRIPTION
**Pull request by t3azr**
[x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

Use useMenuEntriesState to create the index menu options for events/id/index and organizations/id/index. Add idStr to prevent NaN from breaking urls.

Used useMenuEntriesState to create the menu options for both events and organizations. Changed the id used to create URL routes in useMenuEntriesState . 

When tested using an example organization, redirected to the correct URL asfaik. 

- #870 

Hope this is good!! I think the commit name may have become messed up when trying to commit if you need that fixed, thank you !!

### CODE REVIEW

**Comment by andrewtavis** [comment]
Thanks so much for the PR, @t3azr! Would you want to give the [frontend warnings](https://github.com/activist-org/activist/actions/runs/10203261176/job/28288295441#step:6:1) a quick look? I can also support it needed! 😊

**Comment by t3azr** [comment]
sorry for taking so long !! im pretty sure the only errors were because I forgot to use prettier to format the file , but if there's anything else wrong please contact me !! hopefully these changes have been updated and work !!

File: frontend/composables/useMenuEntriesState.ts [hunk 1]
```diff
@@ -3,8 +3,11 @@ import { IconMap } from "~/types/icon-map";
 const createMenuEntry = (label: string, basePath: string, iconURL: string) => {
   const { locale } = useI18n();
   const router = useRouter();
+  // here, afaik, this is returning NaN as the id of a given organization is alphanumeric -- not a number.
+  // added the raw id which can be used with the routeURL , while maintaining the number version
+  const idStr = router.currentRoute.value.params.id;
   const id = Number(router.currentRoute.value.params.id);
```
Comment (andrewtavis) [inline frontend/composables/useMenuEntriesState.ts:9]: Ya this is a holdover from prior setups where the route URL was an integer. Nice catch! Let's just set id to your value - without the `Number()` :)

**Review by andrewtavis — APPROVED** [review]
All's working really well, @t3azr! Really appreciate you taking the time to make these changes and that you reacted to the tests as well! Sorry my review was a bit on the late side :) Getting back into things after needing to focus elsewhere 😊 Hope you pick up another issue soon!

**Pull request merged**